src/main/kotlin/solve/importer/view/ImporterView.kt

```diff
@@ -1,12 +1,27 @@
 package solve.importer.view
 
 import javafx.geometry.Insets
+import javafx.stage.Stage
 import tornadofx.borderpane
 import tornadofx.View
+import tornadofx.getChildList
 
 class ImporterView : View("Choose working directory") {
+
+    private val buttonView: ButtonView by inject()
+
+    override fun onDock() {
+        currentWindow?.setOnCloseRequest {
+            buttonView.root.children[1].getChildList()!![1].isDisable = true
```

Comment:
wtf?? do not use such hacks))

Reply:
and what is other way?

Reply:
You can bind isDisable property of the component to a property "hasAnyErrors", use MVC.
Current code has the following disadvantages:
1. What is ` buttonView.root.children[1]`??
2. What if `buttonView.children` will be changed in the future and the control you get here will be on [2] or on [0] position?
3. What is `buttonView.root.children[1].getChildList()!![1]`??
4. Why it is done on every close request?